Sonic/ObjectSystem/Manager/SetObjectManager.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+namespace Sonic
+{
+    class CUserIDGroupCategoryManager;
+    class CParameterBankManager;
+    class CSetLayerManager;
+    class CSetObjectEventManager;
+    class CSetObjectManager
+    {
+    public:
+        class CSetObjectManagerMember
```

Comment:
This class likely is just named CMember

Reply:
Renamed to CMember, I thought there could've been issues by calling it that 😅